lib/hooks/queries/useIdentities.ts

```diff
@@ -0,0 +1,75 @@
+import { useQuery } from "@tanstack/react-query";
+import { isRpcSdk } from "@zeitgeistpm/sdk";
+import { Judgement, UserIdentity } from "lib/types/user-identity";
+import { useSdkv2 } from "../useSdkv2";
+
+export const identitiesRootKey = "identities";
+
+export const useIdentities = (addresses?: string[]) => {
+  const [sdk, id] = useSdkv2();
+
+  const query = useQuery(
+    [id, identitiesRootKey, addresses],
+    async () => {
+      if (addresses && isRpcSdk(sdk)) {
+        const userIdentities: (UserIdentity | null)[] = [];
+        const res = await sdk.api.queryMulti(
+          addresses.map((address) => [
+            sdk.api.query.identity.identityOf as any,
```

Comment:
is it possible to use a more defined type instead of `any` ?

Reply:
The api doesn't seem to have a type for this for some reason. I could write one, though the substrate types are a bit of a pain to write out manually and it wouldn't bring us much additional safety.